### PR TITLE
Update Clips4Sale.yml

### PR DESCRIPTION
**Pull request by Belleyy**
New look for the site..
Added the keyword in the tags
#123

### CODE REVIEW

**Comment by peolic** [comment]
Uhh, I guess not all studios got the update?

File: scrapers/Clips4Sale.yml [hunk 1]
```diff
@@ -6,40 +6,56 @@ sceneByURL:
...
+              - regex: \s*Optimum
+                with:
+              - regex: \s*OPTIMUM
+                with:
+              - regex: \s*WMV
+                with:
+              - regex: \s*\(HD-1080p\)
+                with:
+              - regex: \s*\(HD-720p\)
+                with:
```
Comment (peolic) [inline scrapers/Clips4Sale.yml:25]: Good point, we can expand that further.
Example: https://regex101.com/r/CYLeuO/1

**Comment by Belleyy** [comment]
> 
> 
> Uhh, I guess not all studios got the update?

Oh yea...
Look like the new layout is `https://www.clips4sale.com/search/clip/'
But old still exist...

**Comment by peolic** [comment]
Ohhh the `search/clip`!
It has been around for at least a few months.
It would be best to have a separate scraper in the same file just for this type of URL since it can be detected.

File: scrapers/Clips4Sale.yml [hunk 2]
```diff
@@ -1,45 +1,87 @@
...
     scene:
-      Title: //div[@data-clipid]/following-sibling::h3
+      Title:
+        selector: //div[@data-clipid]/following-sibling::h3
+        postProcess: &ppStripTitle
+          - replace:
+              # https://regex101.com/r/CYLeuO/8
+              - regex: (?i)[ \t]*((Super )?[SH]D)?[ ,-]*(\b(MP4|OPTIMUM|WMV|MOV|AVI|UHD|[48]K)\b|1080p|720p|480p|(Standard|High) Def(inition)?)+[ \t]*
+                with:
       Details:
```
Comment (peolic) [inline scrapers/Clips4Sale.yml:25]: Sorry, just thought about it, might as well add it.
```suggestion
                with:
              # https://regex101.com/r/C3TgFG/1
              - regex: [ \t]*[\[(][\])][ \t]*|[ -]+$
                with:
      Details:
```

(and so nice you only have to add it once 😄)

Comment (peolic) [reply scrapers/Clips4Sale.yml:25]: oh, wait it's broken

File: scrapers/Clips4Sale.yml [hunk 3]
```diff
@@ -1,45 +1,87 @@
...
       $studio: //span[contains(text(),"From:")]/following-sibling::a
     scene:
-      Title: //div[@data-clipid]/following-sibling::h3
+      Title:
+        selector: //div[@data-clipid]/following-sibling::h3
+        postProcess: &ppStripTitle
+          - replace:
+              # https://regex101.com/r/CYLeuO/8
+              - regex: (?i)[ \t]*((Super )?[SH]D)?[ ,-]*(\b(MP4|OPTIMUM|WMV|MOV|AVI|UHD|[48]K)\b|1080p|720p|480p|(Standard|High) Def(inition)?)+[ \t]*
+                with:
```
Comment (peolic) [inline scrapers/Clips4Sale.yml:24]: This isn't broken.
> Sorry, just thought about it, might as well add it.
```suggestion
              - regex: >-
                  (?i)[ \t]*((Super )?[SH]D)?[ ,-]*(\b(MP4|OPTIMUM|WMV|MOV|AVI|UHD|[48]K)\b|1080p|720p|480p|(Standard|High) Def(inition)?)+[ \t]*
                with:
              # https://regex101.com/r/C3TgFG/2
              - regex: >-
                  [ \t]*[([][])][ \t]*|[ -]+$
                with:
```
> (and so nice you only have to add it once 😄)

**Pull request merged**